web-admin/src/features/projects/user-management/OrganizationItem.svelte

```diff
@@ -41,9 +46,12 @@
         shape="square"
         name={organization}
         count={userGroupMemberUsersCount}
-        isEveryFromText
+        isEveryoneFromText
+        canManage={onMetricsExplorerPage}
```

Comment:
I'm a little confused how this `canManage` is different from the prop `canManage`